test/BitStringAddresses.jl

```diff
@@ -549,7 +549,7 @@ end
         lfs = OccupationNumberFS{6}([1 0 0; 1 1 0])
         @test onr(lfs, LadderBoundaries(2, 3)) == [1 0 0; 1 1 0]
         @test num_occupied_modes(lfs) == length(occupied_modes(lfs)) == 3
-        @test OccupiedModeMap(lfs) == collect(occupied_modes(lfs))
+        @test occupied_mode_map(lfs) == collect(occupied_modes(lfs))
```

Comment:
Hmm I realized we don't have a test like this for `FermiFS` or `BoseFS`. Could you maybe sneak it in somewhere?